Web Dev/Backend/Frameworks/JS based/Express JS.md

```diff
@@ -0,0 +1,114 @@
+## What is  Express.js ❓
+Express.js helps you build web applications by providing a set of tools and features that make it easy to handle HTTP requests and responses, create routes, manage middleware, and integrate with databases. It is a web framework application for 🎨 Node JS
+
+## Why learn Express.js ❓
+First of all, Express.js is easy to learn and use. It is highly flexible and can be used to build a wide range of web applications. Moreover, it is built of top of 🎨 Node JS  making it more fast and efficient for building web applications.
+
+## 🎓 Skills you can Master
+After completing this learning path, you will be knowledgable in:
+- Routing and middleware
+- Authentication and Authorization
+- Working with databases
+- Automated testing

```

Comment:
add a section about error handlers , error handling is probbaly the most important thing on a api